README.md

```diff
@@ -1 +1,69 @@
-# my-portfolio-site-2
\ No newline at end of file
+# my-portfolio-site-1
+
```

Comment:
- [ ] OPTIONAL 
Your ReadMe file appears not very well organized and very descriptive. Please add a more descriptive README file to your project by using [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project.